docs/articles/install/3-install-with-helm.md

```diff
@@ -38,29 +41,29 @@ helm upgrade --install \
   --create-namespace \
   --namespace testkube \
    -f values.yaml \
-  testkube testkubeenterprise/testkube-enterprise
+  testkube kubeshop/testkube-enterprise
```

Comment:
The Helm Chart is now available under `kubeshop/testkube-enterprise` right?

Reply:
if you want to change this to the kubeshop, then update `helm repo add testkubeenterprise https://kubeshop.github.io/testkube-cloud-charts` to `helm repo add kubeshop https://kubeshop.github.io/testkube-cloud-charts`. And the same for the command `helm show values...`

Reply:
Will this not clash with `helm repo add kubeshop https://kubeshop.github.io/helm-charts`?

Reply:
I've reverted it for now due to my concern in the previous comment.

Reply:
Yes, I would think you can't have two repos with same name. The cloud one should probably not be named kubeshop.